src/lib/label/label.ts

```diff
@@ -99,7 +94,7 @@ export class LabelComponent extends BaseComponent implements ILabelComponent {
   public dynamic: boolean;
 
   @FoundationProperty()
-  public static: boolean;
+  public nonInteractive: boolean;
```

Comment:
We probably missed this from the original PR, but can you add the `declare` keyword to all public properties in this file that use the `@FoundationProperty` decorator?